File: Basic Programs/Problem_14.js
/*--------------------------------
Problem 14: Area and Perimeter of Rectangle

Write a program that takes length and width of a rectangle as input and calculates both its area and
perimeter, then displays both results.
--------------------------------*/

const length = 18;
const width = 9;

const areaOfRectangle = length * width;
const perimeterOfRectangle = 2 * areaOfRectangle;

console.log(`Area of Rectangle is: ${areaOfRectangle}`);
console.log(`Perimeter of Rectangle is: ${perimeterOfRectangle}`);

/* OUTPUT:
Area of Rectangle is: 162
Perimeter of Rectangle is: 324
*/

File: shivam-chaurasiya/Practice Problems/Level 1/01 Beginner Programs/Problem14.py
# Area and Perimeter of Rectangle

# Area of rectangle = l × b 
# Perimeter of rectangle = 2(l + b)

def rectangle(l, b):
    aor = l*b
    por = 2*(l*b)
    return f'The area of rectangle is: {aor} and The Perimeter of rectangle is: {por}'

l = int(input('Enter the length of rectangle:'))
b = int(input('Enter the width of rectangle:'))
print(rectangle(l, b))